### ISSUE TITLE
Can't re-authorize

### ISSUE THREAD

**Issue by TheNastyness** [grzegorz914/homebridge-xbox-tv#204]
I've been having issues with the plugin for some time.  I've tried removing the plugin and re-installing in an attempt to try to resolve the problem, but can't get it fixed.

Homebridge logs show the following error message:  Error [ERR_UNHANDLED_ERROR]: Unhandled error. ('No oauth token found. Use authorization manager first.')

When I try to use the Authorization Manager to reauthenticate, the following message is returned:  Console authorized and activated. To start new process please clear Web API Token first.

When I try to clear the token and start the process again, no pop-up window appears, it simply says:  Console authorized and activated. To start new process please clear Web API Token first.

Basically, I'm stuck in an endless loop of clearing the token and starting a new authorization.  Every time I do, the plugin says that my console is authorized, but I still get the error message in homebridge logs.

**Comment by grzegorz914** [comment]
look into Homebridge/xboxTv folder and try to remove the token file manually then start authorization again

**Comment by TheNastyness** [comment]
I'm running homebridge on a Mac mini, do you happen to know where this folder would be located on a Mac?

**Comment by TheNastyness** [comment]
Did not work, it's as if my Xbox details are being cached by the plugin, as the authToken file reappears in the folder.  I never get a prompt or redirect when I try to clear the token and reauthorize.

**Comment by grzegorz914** [comment]
It’s appear but only empty token

**Comment by TheNastyness** [comment]
Yes.  See attached
![Screen Shot 2024-08-18 at 12 36 46 PM](https://github.com/user-attachments/assets/a5b08e02-870c-4f6d-9add-c3d2fd0f9312)

**Comment by grzegorz914** [comment]
Ok, it is bug, same happen now for me, I will investigate

**Comment by TheNastyness** [comment]
Can I input the authorization manually by entering the authorization URL into my web browser?

**Comment by grzegorz914** [comment]
Fixed in 3.0.2
